ViewLayer/RaceModeDisplay/RaceModeDashboardView.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <QObject>
+#include <QLabel>
+
+#include "../../DataLayer/MpptData/Mppt.h"
+#include "../../DataLayer/MotorFaultsData/ErrorFlags.h"
+#include "../../DataLayer/MotorFaultsData/LimitFlags.h"
+#include "../../PresenterLayer/BatteryPresenter/BatteryPresenter.h"
+#include "../../PresenterLayer/BatteryFaultsPresenter/BatteryFaultsPresenter.h"
+#include "../../PresenterLayer/DriverControlsPresenter/DriverControlsPresenter.h"
+#include "../../PresenterLayer/KeyMotorPresenter/KeyMotorPresenter.h"
+#include "../../PresenterLayer/LightsPresenter/LightsPresenter.h"
+#include "../../PresenterLayer/MpptPresenter/MpptPresenter.h"
+#include "../../PresenterLayer/MotorDetailsPresenter/MotorDetailsPresenter.h"
+#include "../../PresenterLayer/MotorFaultsPresenter/MotorFaultsPresenter.h"
+#include "../DisplayDashboard/DisplayDashboardUI/I_RaceModeDashboardUI.h"
```

Comment:
Right now it looks like we are including these classes AND forward declaring them. Let's just forward declare and keep the includes in the .cpp
https://stackoverflow.com/questions/4757565/what-are-forward-declarations-in-c